### DISCUSSION TITLE
"DAG mix match task timings issues with the DAG/Grid/timing" that the Run on the most right "Started | 2022-10-19, 14:55:31 UTC" and all elements before are later than that.

### DISCUSSION THREAD

**Discussion by stromal** [apache/airflow#27407] [Q&A]
### Apache Airflow version

2.4.1

### What happened

# "DAG mix match task timings issues with the DAG/Grid/timing" that the Run on the most right "Started | 2022-10-19, 14:55:31 UTC" and all elements before are later than that.

## DAG mix match task timings issues with the DAG/Grid/timing

- Even newly run task appears before the oldest task that is on the right.

![Screenshot 2022-10-19 162447](https://user-images.githubusercontent.com/19979901/196737134-15d9793a-98f3-47e1-9468-94bedd202a73.jpg)

### Tasks left to right (all of these have been triggered by me and did not run on their scheduled time: 08:16:00 UTC & 11:12:00 UTC)

1.
- official scheduled time to Run: 2022-10-19, 08:16:00 UTC
- Started | 2022-10-19, 14:58:33 UTC
- Ended | 2022-10-19, 14:58:36 UTC

2.
- official scheduled time to Run: 2022-10-19, 08:16:00 UTC
- Started | 2022-10-19, 15:10:52 UTC
- Ended | 2022-10-19, 15:10:56 UTC

3.
- official scheduled time to Run: 2022-10-19, 08:16:00 UTC
- Started | 2022-10-19, 15:13:20 UTC
- Ended | 2022-10-19, 15:13:23 UTC

4.
- official scheduled time to Run: 2022-10-19, 08:16:00 UTC
- Started | 2022-10-19, 15:14:10 UTC
- Ended | 2022-10-19, 15:14:13 UTC

5. Latest Run
- official scheduled time to Run: 2022-10-19, 08:16:00 UTC
- Started | 2022-10-19, 15:14:22 UTC
- Ended | 2022-10-19, 15:14:25 UTC

7. First run
- official scheduled time to Run: 2022-10-19, 11:12:00 UTC
- Started | 2022-10-19, 14:55:31 UTC
- Ended | 2022-10-19, 14:55:34 UTC

#### What you think should happen instead

Start the file run at the specific period even if it will fail

#### Operating System

Ubuntu 20.04 LTS

#### Versions of Apache Airflow Providers

IMAGE extended: ```apache/airflow:2.4.1-python3.8```` 

#### Deployment

Other Docker-based deployment

#### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

#### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by stromal** [comment]
@potiuk

another example



![2022 11 07  Airflow error_2](https://user-images.githubusercontent.com/19979901/200363713-1cd5f78d-d27e-490d-a278-ef1e61a68071.png)


RUN TIMES LEFT TO RIGHT
1.
<html>
<body>


Duration | 00:00:08
-- | --
Started | 2022-11-07, 15:58:59 GMT
Ended | 2022-11-07, 15:59:07 GMT


</body>
</html>

2.
<html>
<body>


Duration | 00:00:05
-- | --
Started | 2022-11-07, 16:13:37 GMT
Ended | 2022-11-07, 16:13:43 GMT


</body>
</html>

3.
<html>
<body>


Duration | 00:00:28
-- | --
Started | 2022-11-07, 16:24:22 GMT
Ended | 2022-11-07, 16:24:51 GMT


</body>
</html>

4.
<html>
<body>


Duration | 00:00:06
-- | --
Started | 2022-11-07, 15:47:49 GMT
Ended | 2022-11-07, 15:47:56 GMT


</body>
</html>

5.

<html>
<body>


Duration | 00:00:07
-- | --
Started | 2022-11-07, 15:55:58 GMT
Ended | 2022-11-07, 15:56:06 GMT


</body>
</html>

**Comment by potiuk** [comment]
I do not uderstand what you are asking for - the way how you abuse fonts and formulate your questions is difficult to understand.
But from what I see you should go back and re-read the docs of airflow when it comes to triggered dags - there is nothing ilike "offiicial" schedule for triggered dags. Those have no "scheduie" - they will run when triggered and scheduie is ignored for them. You are likely looking for a problem that only exists for your bad assumptions (but I am guessing).

Also please do not tag me in your questions. I might be not the right pesron to answer you as I am more and more weary of having to explain all the concepts of airflow to you (that you can find in the docs) and I think it is better if ohters will start answering your qustions. (unless they also decide to skip it).
